app/actions/remote/post.ts

```diff
@@ -552,3 +552,36 @@ export const togglePinPost = async (serverUrl: string, postId: string) => {
         return {error};
     }
 };
+
+export const markPostAsUnread = async (serverUrl: string, postId: string, channelId: string) => {
+    const database = DatabaseManager.serverDatabases[serverUrl]?.database;
+    if (!database) {
+        return {error: `${serverUrl} database not found`};
+    }
+    let client;
+    try {
+        client = NetworkManager.getClient(serverUrl);
+    } catch (error) {
+        return {error};
+    }
+    try {
+        const userId = await queryCurrentUserId(database);
+        const myChannel = await queryMyChannel(database, channelId);
+
+        if (myChannel) {
+            client.markPostAsUnread(userId, postId);
+
+            await database.write(async () => {
+                await myChannel.update((c) => {
+                    c.manuallyUnread = true;
```

Comment:
I think more things have to be updated in myChannel. Check the websocket event that handles unreads.

Reply:
Thanks.  I missed the unread count badge.  I'm adding it